Syntaxes/Swift.tmLanguage

```diff
@@ -4508,14 +4513,113 @@
 								</dict>
 							</dict>
 							<key>end</key>
+							<string>"""#(#*)</string>
+							<key>endCaptures</key>
+							<dict>
+								<key>0</key>
+								<dict>
+									<key>name</key>
+									<string>punctuation.definition.string.end.swift</string>
+								</dict>
+								<key>1</key>
+								<dict>
+									<key>name</key>
+									<string>invalid.illegal.extra-closing-delimiter.swift</string>
+								</dict>
+							</dict>
+							<key>name</key>
+							<string>string.quoted.double.block.raw.swift</string>
+							<key>patterns</key>
+							<array>
+								<dict>
+									<key>match</key>
+									<string>\G.+(?=""")|\G.+</string>
+									<key>name</key>
+									<string>invalid.illegal.content-after-opening-delimiter.swift</string>
+								</dict>
+								<dict>
+									<key>match</key>
+									<string>\\#\s*\n</string>
+									<key>name</key>
+									<string>constant.character.escape.newline.swift</string>
+								</dict>
+								<dict>
+									<key>include</key>
+									<string>#raw-string-guts</string>
+								</dict>
+								<dict>
+									<key>comment</key>
+									<string>Allow \("""...""") to appear inside a block string</string>
+									<key>match</key>
+									<string>\S((?!\\#\().)*(?=""")</string>
+									<key>name</key>
+									<string>invalid.illegal.content-before-closing-delimiter.swift</string>
+								</dict>
+							</array>
+						</dict>
+						<dict>
+							<key>begin</key>
+							<string>(##*)"""</string>
```

Comment:
This should probably have been `(##+)"""`. It still works with `*` but only because of previous rule.